src/server_manager/web_app/digitalocean_server.ts

```diff
@@ -424,7 +424,8 @@ function getInstallScript(
   const sanitizezedAccessToken = sanitizeDigitaloceanToken(accessToken);
   return '#!/bin/bash -eu\n' +
       `export DO_ACCESS_TOKEN=${sanitizezedAccessToken}\n` +
-      (image ? `export SB_IMAGE=${image}\n` : '') +
+      // Set watchtower to 30 second refresh rate when overriding image.
+      (image ? `export SB_IMAGE=${image}\nexport WATCHTOWER_REFRESH_SECONDS=30\n` : '') +
```

Comment:
I'll add a TODO, but this actually came up in the past and we didn't address it because we weren't convinced of the threat model (it seems like users who start the electron app via the command line could inject commands into their own script, but we haven't seen any way for an outside attacker to inject commands).  In any case I agree it's a bit of a code smell